server/server.go

```diff
@@ -88,6 +91,25 @@ func (s *Server) clientForRequest(req parseRequest) (protocol.ProtocolServiceCli
 	return newClient(req.ServerURL)
 }
 
+func (s *Server) LoadGist(ctx *gin.Context) {
```

Comment:
I decided to keep it for now. Maybe it will be useful in near future. If not, we always can remove it.